core/src/apps/ethereum/sign_tx.py

```diff
@@ -99,16 +105,89 @@ async def sign_tx(
     return result
 
 
-async def handle_erc20(
+async def confirm_tx_data(
+    msg: MsgInSignTx,
+    defs: Definitions,
+    to_address: bytes,
```

Comment:
very nit but please rename `to_address` and `address_to` in all cases to `address_bytes` so that this is consistent across the whole file.
(`address_bytes` is not _the_ best name but it was here before)

Reply:
fixed in rebase